File: server-scala-play-server/app/model/ContactResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ContactResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class ContactResponse(
  firstName: Option[String],
  middleName: Option[String],
  lastName: Option[String],
  prefix: Option[String],
  suffix: Option[String],
  title: Option[String],
  contactInfo: Option[ContactInfoResponse]
)

object ContactResponse {
  implicit lazy val contactResponseJsonFormat: Format[ContactResponse] = Json.format[ContactResponse]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/ContactResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class ContactResponse 
{
    public string FirstName { get; set; }
    public string MiddleName { get; set; }
    public string LastName { get; set; }
    public string Prefix { get; set; }
    public string Suffix { get; set; }
    public string Title { get; set; }
    public ContactInfoResponse ContactInfo { get; set; }
}


